azure/azure_style.yaml

```diff
@@ -52,3 +52,17 @@ jobs:
           cd ${{ parameters.REPO_NAME }}
           pip install black==20.8b1
           black . --check --diff -l 120 --target-version py37 --target-version py38
+
+  - job: isort
+    pool:
+      vmImage: "ubuntu-20.04"
+    continueOnError: ${{ parameters.IGNORE_ISORT }}
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.8"
+      - script: |
+          cd ${{ parameters.REPO_NAME }}
+          cp ../.github/.isort.cfg .
```

Comment:
The .github repo needs to be checked out separately. For `flake8` we have:
```
- checkout: self
- checkout: azure_template
```

Reply:
Oh right, I somehow missed those.